src/assets/style/theme.js

```diff
@@ -0,0 +1,61 @@
+export const Breakpoints = {
+  xs: '640px',
+  sm: '830px',
+  md: '1100px',
+  lg: '1500px'
+}
+
+const Colors = {
+  primary: '#c089A8',
+  primary_light: '#CC91B2',
+  primary_dark: '#805B6F',
+  secondary: '#f8cbAe',
+  secondary_light: '#DEB69B',
+  secondary_dark: '#B89781',
+  gray: '#dfe6e9',
+  gray_15: '#bdc3c7',
+  gray_50: '#636e72',
+  light: '#ffffff',
+  dark: '#2c2b2b',
+  facebook: '#3b5998',
+  red: '#d63031'
+}
+
+const FontSize = {
```

Comment:
Don't think you need these at all as you're not saving any space really by having them